tests/usbus_hid/Makefile

```diff
@@ -0,0 +1,12 @@
+BOARD ?= nrf52840dk
+
+include ../Makefile.tests_common
+
+USEMODULE += usbus_hid
+
+DISABLE_MODULE += auto_init_usbus
+
+USB_VID=${USB_VID_TESTING}
```

Comment:
Maybe even better `USB_VID ?= $(USB_VID_TESTING)` to allow change during `make` call.